libcxx/include/__algorithm/fill.h

```diff
@@ -21,23 +21,64 @@ _LIBCPP_BEGIN_NAMESPACE_STD
 
 // fill isn't specialized for std::memset, because the compiler already optimizes the loop to a call to std::memset.
 
-template <class _ForwardIterator, class _Tp>
+template <
+    class _ForwardIterator,
```

Comment:
Not attached: Please add a release note for this improvement.

Reply:
Added release note.